File: C++/day15.cpp
      Node* insert(Node *head,int data)
      {
          if (head == NULL)
          {
              head = new Node(data);
          }
          else
          {
              Node *n, *h=head;
              n = new Node(data);
              while (h->next!=NULL)
                  h = h->next;
              h->next=n;
          }
          return head;
      }

File: PYTHON2/day15.py
def insert(self,head,sata): 
        if head==None:
            head=Node()
            head.data=sata
        else:
            start=head
            while start.next :start=start.next
            start.next=Node()
            start=start.next
            start.data=sata
        return head
